eclass/nvidia-driver.eclass

```diff
@@ -212,21 +221,21 @@ nvidia-driver_check_kernel() {
 		ewarn "which are limited to the following kernels:"
```

Comment:
I guess we should make this more generic for gentoo-kernel? virtual/linux-sources or virtual/dist-kernel?

Reply:
`virtual/linux-sources` includes stuff like `sys-kernel/zen-sources`, do we want to promote that?

Reply:
Maybe just list gentoo-kernel as another common choice. I wouldn't bother with vanilla since it's not security supported and we don't do heavy patching anyway.